projects/openexr/openexr_exrenvmap_fuzzer.cc

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
```

Comment:
Sorry if this was already asked elsewhere, but is there a plan for moving the fuzzer source upstream eventually? It's fine for now, but simplifies maintenance since updates won't have to be approved by the OSS-Fuzz team.

Reply:
As I'm an intern this summer, my objective is to get as many projects initially integrated as possible. My host, @alsophian, and I have been keeping track of the fuzzers I need to integrate, and because we are steadily approaching the end of the list, moving these fuzzers upstream within the duration of my internship is not out of the question. That said, I'm not sure we'll be able to get to it given the other work I have on my plate.

@alsophian care to chime in?

Reply:
Ravi's got it right here.  We're building a good relationship with the OpenEXR maintainers, so if Ravi doesn't have time to upstream these fuzzers this summer, I'll work with them on a best-effort basis to eventually move them upstream.